### PR TITLE
Restructure `dispatch` macro related exports

### PR DESCRIPTION
**Pull request by juangirini**
Partial for https://github.com/paritytech/polkadot-sdk/issues/172

### CODE REVIEW

**Review by sam0x17 — APPROVED** [review]

**Review by bkchr — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+**/target/
+.vscode
+.DS_Store
```
Comment (liamaharon) [inline .gitignore:3]: 👀

File: substrate/frame/support/src/dispatch_context.rs [hunk 2]
```diff
@@ -26,11 +26,11 @@
 //!
 //! # FRAME integration
 //!
-//! The FRAME macros implement [`UnfilteredDispatchable`](crate::traits::UnfilteredDispatchable) for
-//! each pallet `Call` enum. Part of this implementation is the call to [`run_in_context`], so that
-//! each call to
+//! The FRAME macros implement
+//! [`UnfilteredDispatchable`](frame_support::traits::UnfilteredDispatchable) for each pallet `Call`
```
Comment (liamaharon) [inline substrate/frame/support/src/dispatch_context.rs:30]: cool, I had no idea you could link with []() 🤯

**Review by liamaharon — CHANGES_REQUESTED** [review]
LGTM, just need to remove the .gitignore

**Review by liamaharon — APPROVED** [review]
Disregard prev. review, I see now it was a merge conflict

File: polkadot/runtime/common/src/claims.rs [hunk 3]
```diff
@@ -1470,7 +1470,7 @@ mod benchmarking {
 	use super::*;
 	use crate::claims::Call;
 	use frame_benchmarking::{account, benchmarks};
-	use frame_support::dispatch::UnfilteredDispatchable;
```
Comment (ggwpez) [inline polkadot/runtime/common/src/claims.rs:1473]: Now we just need 1 MR instead of three, what a breeze :relieved:

**Review by ggwpez — APPROVED** [review]

**Comment by ggwpez** [comment]
bot fmt

**Comment by juangirini** [comment]
bot rebase

**Review by sam0x17 — APPROVED** [review]

**Review by acatangiu — APPROVED** [review]

**Review by ordian — APPROVED** [review]

**Pull request merged**